apps/admin/src/app/report/[id]/actions.tsx

```diff
@@ -59,7 +59,7 @@ export function ReportActions({ report }: ReportActionsProps) {
         title: 'Success',
         description: <p>Comment deleted successfully</p>,
       });
-      router.push('/admin');
+      router.push('/');
```

Comment:
do we need to append querystrings for correct tab?

Reply:
mm probably.

Reply:
looks like so 

```ts
              <Link href="/?tab=tracks">
                <Button variant="ghost">Cancel</Button>
              </Link>
```